src/components/InstanceEditor.svelte

```diff
@@ -15,6 +15,27 @@
 	let fileInput: HTMLInputElement;
 	let colourInput: HTMLInputElement;
 
+	let isDragging = false;
```

Comment:
I'm not sure I understand the utility of this variable. You don't seem to access its value anywhere. Maybe it (this line and all the lines that reference it) can just be removed entirely?